pyproject.toml

```diff
@@ -50,7 +50,7 @@ dependencies = [
   "importlib-metadata; python_version < '3.8'",
   "torch>1.9,<1.13",
   "requests",
-  "pydantic==1.9.2",
+  "pydantic",
```

Comment:
Maybe worth pinning `pydantic<2` to avoid having the same problems all over again when Pydantic 2.0 comes out? @masci

Reply:
IMHO, I don't think it's necessary. 

Here you can see what will be removed: https://pydantic-docs.helpmanual.io/blog/pydantic-v2/#removed-features-limitations.

The dataclasses will only have internal changes, not on the external interface. https://pydantic-docs.helpmanual.io/blog/pydantic-v2/#features-remaining